src/wp-admin/customize.php

```diff
@@ -186,10 +186,10 @@
 			$compatible_php = is_php_version_compatible( $wp_customize->theme()->get( 'RequiresPHP' ) );
 			$fse_safe       = true;
 
-			// Check if the theme requires the FSE to work correctly.
+			// Check if the theme requires the Gutenberg plugin to work correctly.
 			$theme_tags = $wp_customize->theme()->get( 'Tags' );
 
-			if ( ! empty( $theme_tags ) && in_array( 'full-site-editing', $theme_tags, true ) && ! function_exists( 'gutenberg_is_fse_theme' ) ) {
+			if ( ! empty( $theme_tags ) && in_array( 'full-site-editing', $theme_tags, true ) && ! function_exists( 'gutenberg_is_block_theme' ) ) {
```

Comment:
This checks for a function within Gutenberg itself. Because `guteberg_is_fse_theme()` still exists in the plugin for versions <= 10.9.0 (assuming 10.10.0 includes the changes in the PR referenced in the issue description), this would have to check for both functions to be backwards and forwards compatible.